Disc/Networking/APIClient.swift

```diff
@@ -2,6 +2,7 @@ import Swish
 
 /// An API client for Passport.
 public struct APIClient {
+    static let auth = Swish.APIClient()
     let client = Swish.APIClient()
```

Comment:
I thought about just making `client` `static` instead, but thought that might be weird. I'm not sure.

It's an implementation detail anyway and not exposed as part of the API.